src/Sarif/CodeGenHints.json

```diff
@@ -81,6 +89,16 @@
       }
     }
   ],
+  "FileData.LastModifiedTime": [
+    {
+      "kind": "AttributeHint",
+      "arguments": {
+        "namespaceName": "Newtonsoft.Json",
```

Comment:
>Newtonsoft.Json [](start = 26, length = 15)

@lgolding, another jschema bug, the namespaceName element doesn't appear to be honored by the code emit for this kind of attribute.